jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -982,31 +991,19 @@ protected void startContext() throws Exception
     public void contextInitialized() throws Exception
     {
         // Call context listeners
-        switch (_contextStatus)
+        if (_contextStatus == ContextStatus.NOTSET)
         {
-            case NOTSET:
+            _contextStatus = ContextStatus.INITIALIZED;
```

Comment:
Probably ok to move this here, but being nitpicky, the context has not actually been initialized until _after_ the SCLs have been called.

Reply:
It is moving this that means we can correctly detect and throw ISE if another listener is added.  It is not used other than in lifecycle stuff and is not exposed externally.